main/acle.md

```diff
@@ -2265,6 +2268,15 @@ is implementation-defined.
 The generic itanium C++ ABI, which we use in AArch64, already handles
 arrays with arbitrarily aligned elements
 
+## Scalable Vector Extension procedure call standard
+
+On SVE enabled AArch64 targets, the AAVPCS allows for procedure calls to
+use the SVE calling convention. ACLE allows this to be selectable per function:
```

Comment:
I think you should expand this making it explicit that the mechanism enforces the condition that triggers the behavior for SVE function (the one that says "If a subroutine takes
at least one argument in scalable vector registers or scalable predicate
registers, or if it is a function that returns results in such registers")

It would also be useful to have references to the ABI document / version / section that describe this behavior.

Reply:
I've added an expanded description. I'm assuming we want to reference [here](https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst#scalable-vector-registers), but I'm struggling to work it into this paragraph without it sounding weird. I was thinking along the lines of "For more information see abi-aa section 6.3.3.". I'm not sure if that's reasonable?